bindings/python/examples/client/06_simple_block.py

```diff
@@ -10,5 +10,6 @@
 client = Client(nodes=[node_url])
 
 # Create and post a block without payload
-block = client.build_and_post_block()
+# TODO: have a way in the bindings to send an empty block
```

Comment:
Want to record it in the issue? Resolve if not.

Reply:
Added `Make it possible to send blocks with types and client/secret manager`